python_modules/dagster/dagster/_core/definitions/assets_job.py

```diff
@@ -317,18 +310,19 @@ def _key_for_asset(asset: Union[AssetsDefinition, SourceAsset]) -> AssetKey:
 
 
 def _get_blocking_asset_check_output_handles_by_asset_key(
-    assets_defs_by_node_handle, asset_checks_defs_by_node_handle
+    asset_checks_by_node_handle, asset_checks_defs_by_node_handle
```

Comment:
does this change the type here?

I think this name change is confusing, since it would look like it changes the type. I would assume this is because all of these `AssetsDefinitions` only have checks defined or something crazy like that?